generators/vsix-vs-win/BotBuilderVSIX-V4/Properties/Resources.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace BotBuilderVSIX.Properties {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
```

Comment:
I'm not clear on this, as it seems it would require VS 2019 to work properly. We're not ready yet to abandon running this on VS2017, if nothing else as the VSIX is built and signed though our Azure Devops Build Server, which I don't think has a VS 2019 Sku on it yet.